### PR TITLE
8279164: Disable TLS_ECDH_* cipher suites

### PR DESCRIPTION
**Pull request by seanjmullan**
This change will disable TLS_ECDH_* cipher suites by default. These cipher suites do not preserve forward secrecy and are rarely used in practice. See the CSR for more details and rationale.

Users will still be able to enable the suites (at their own risk) by removing "ECDH" from the `jdk.tls.disabledAlgorithms` security property.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change requires a CSR request to be approved

### Issues
 * [JDK-8279164](https://bugs.openjdk.org/browse/JDK-8279164): Disable TLS_ECDH_* cipher suites
 * [JDK-8296300](https://bugs.openjdk.org/browse/JDK-8296300): Disable TLS_ECDH_* cipher suites (**CSR**)

### Reviewers
 * [Xue-Lei Andrew Fan](https://openjdk.org/census#xuelei) (@XueleiFan - **Reviewer**) ⚠️ Review applies to [897f5603](https://git.openjdk.org/jdk/pull/10969/files/897f560340c1d759b6b7f6a9335568f4455efe3e)
 * [Rajan Halade](https://openjdk.org/census#rhalade) (@rhalade - **Reviewer**) ⚠️ Review applies to [897f5603](https://git.openjdk.org/jdk/pull/10969/files/897f560340c1d759b6b7f6a9335568f4455efe3e)
 * [Bradford Wetmore](https://openjdk.org/census#wetmore) (@bradfordwetmore - **Reviewer**) ⚠️ Review applies to [0aa21bdc](https://git.openjdk.org/jdk/pull/10969/files/0aa21bdc3b2261f87b7c8e54b024dc0ac677a263)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk pull/10969/head:pull/10969` \
`$ git checkout pull/10969`

Update a local copy of the PR: \
`$ git checkout pull/10969` \
`$ git pull https://git.openjdk.org/jdk pull/10969/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 10969`

View PR using the GUI difftool: \
`$ git pr show -t 10969`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/10969.diff">https://git.openjdk.org/jdk/pull/10969.diff</a>

</details>

### CODE REVIEW

**Review by XueleiFan — APPROVED** [review]
Looks good to me.

**Comment by ecki** [comment]
Should probably also be mentioned as planned on the Ora crypto roadmap or will it only be added at release time?

**Comment by seanjmullan** [comment]
> Should probably also be mentioned as planned on the Ora crypto roadmap or will it only be added at release time?

Eventually. This is to first get this change into a feature release.

**Review by rhalade — APPROVED** [review]

File: test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java [hunk 1]
```diff
@@ -82,22 +82,6 @@ public class CheckCipherSuites {
         "TLS_DHE_RSA_WITH_AES_128_CBC_SHA256",
         "TLS_DHE_DSS_WITH_AES_128_CBC_SHA256",
 
-        // AES_256(GCM) - not forward screcy
```
Comment (bradfordwetmore) [inline test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java:85]: Can you please fix the rest of the"screcy" typos in this file?

**Review by bradfordwetmore — APPROVED** [review]
LGTM other than the typos.

File: test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java (hunk 1 above)
Comment (seanjmullan) [reply test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java:85]: Fixed.

Comment (bradfordwetmore) [reply test/jdk/javax/net/ssl/sanity/ciphersuites/CheckCipherSuites.java:85]: Thanks.

**Comment by bradfordwetmore** [comment]
Thanks.

**Pull request closed without merging**